mllib/src/main/scala/org/apache/spark/ml/param/params.scala

```diff
@@ -249,6 +249,16 @@ object ParamValidators {
   def arrayLengthGt[T](lowerBound: Double): Array[T] => Boolean = { (value: Array[T]) =>
     value.length > lowerBound
   }
+
+  /** Check if more than one param in a set of exclusive params are set. */
+  def checkExclusiveParams(model: Params, params: String*): Unit = {
```

Comment:
@hhbyyh Is this the common helper methods you want for checking in/out single/multiple columns ?